src/test/java/core/basesyntax/service/impl/WriteFileServiceImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.WriteFileService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WriteFileServiceImplTest {
+    private static WriteFileService writeFileService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        writeFileService = new WriteFileServiceImpl();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void write_WrongPathToFile_NotOk() {
+        String report = "fruit,quantity"
+                + System.lineSeparator()
+                + "banana,14"
+                + System.lineSeparator()
+                + "apple,10"
+                + System.lineSeparator()
+                + "orange,67"
+                + System.lineSeparator();
+        writeFileService.write(report, "src/test/java/resources/EmptyInput.csv");
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void write_EmptyReport_NotOk() {
+        String report = "";
+        writeFileService.write(report, "src/test/java/resources/ToFile.csv");
```

Comment:
```suggestion
        writeFileService.write(report, "src/test/resources/ToFile.csv");
```
папки resources повинна знаходитись в папці test. перевір щоб всі шляхи до файлів в твоєму проекті були правильними (де це очікується).  зараз в цьому тесті ти відловлюєш RuntimeException пов'язану з неправильним шляхом до файлу, хоча очікуєш іншу помилку.